lib/transports/csv.js

```diff
@@ -125,7 +125,8 @@ class csv extends base {
       delimiter: this.parent.options.csvDelimiter,
       rowDelimiter: this.parent.options.csvRowDelimiter,
       quote: this.parent.options.csvQuoteChar,
-      escape: this.parent.options.csvEscapeChar
+      escape: this.parent.options.csvEscapeChar,
+      includeEndRowDelimiter: this.parent.options.csvEscapeChar
```

Comment:
copy-paste bug. should be `this.parent.options..csvIncludeEndRowDelimiter`

Reply:
Sorry for that... end of week...
It's fixed.